python-checks/src/test/resources/checks/hotspots/csrfDisabledCheck/flask/flaskExempt.py

```diff
@@ -11,3 +12,8 @@ def exemptExample():
   #            ^^^^^^
   def csrftestpost():
       pass
+
+  @app.route('/csrftest1/', methods=['POST'])
```

Comment:
It could be nice to have a compliant test showing that the `exempt` is what triggers the issue, such as:
```
  @app.route('/csrftest1/', methods=['POST'])
  @exported_csrf_protect.something_else
  def compliant_from_reexport():
    pass
```